_includes/manuals/1.7/4.3.7_smartproxy_puppet.md

```diff
@@ -112,6 +112,11 @@ The puppetssh provider uses SSH to connect to the client using SSH keys and run
 #:puppetssh_keyfile: /etc/foreman-proxy/id_rsa
 </pre>
 
+And then add a sudoers rule:
```

Comment:
perhaps add "if puppetssh_sudo is enabled"?  (I assume it won't sudo to root otherwise)